string state
string substate_normal
string substate_find
string room_info_0
string room_info_1
string room_info_2
string room_info_3
string room_info_4
string room_info_5
string sleep_timer_info
string last_command
string game_info